dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/TimeBasedLogRolloverTest.java

```diff
@@ -427,24 +425,10 @@ private static void checkForRolledLogsAtTime(Calendar cal, boolean trace) throws
 
         File messagesLog = new File(getLogsDirPath(), messagesLogName);
 
-        // If exact file doesn't exist, look for files within the same minute (30-second tolerance)
+        // Use the helper method
         if (!messagesLog.exists()) {
             LOG.info("Exact file not found, searching with tolerance...");
-            String expectedMinute = date.substring(0, date.length() - 5); // Remove ".00.0" to get minute pattern
-
-            File[] files = logsDir.listFiles();
-            if (files != null) {
-                for (File file : files) {
-                    String fileName = file.getName();
-                    if (fileName.startsWith(MESSAGES_LOG_PREFIX) &&
-                        fileName.endsWith(LOG_EXT) &&
-                        fileName.contains(expectedMinute)) {
-                        LOG.info("Found matching file with tolerance: " + fileName);
-                        messagesLog = file;
-                        break;
-                    }
-                }
-            }
+            messagesLog = findFileWithTolerance(MESSAGES_LOG_PREFIX, date, LOG_EXT);
```

Comment:
Consider passing in the `logsDir` into the method and removing the `LOG_EXT`, as it's a global var.